automatic/kubernetes-kompose/kubernetes-kompose.nuspec

```
@@ -5,31 +5,37 @@
     <id>kubernetes-kompose</id>
     <version>1.22.0</version>
     <packageSourceUrl>https://github.com/chocolatey-community/chocolatey-coreteampackages/tree/master/automatic/kubernetes-kompose</packageSourceUrl>
-    <owners>chocolatey-community,Andrei Bejenaru</owners>
+    <owners>chocolatey-community,dgalbraith,Andrei Bejenaru</owners>
     <title>Kompose: Go from Docker Compose to Kubernetes</title>
-    <authors>Kubernetes authors</authors>
+    <authors>The Kubernetes Authors</authors>
     <projectUrl>http://kompose.io</projectUrl>
     <iconUrl>https://cdn.jsdelivr.net/gh/chocolatey-community/chocolatey-coreteampackages@e4e529ccc804ac6fac259a658e395e38b42b24f3/icons/kubernetes-kompose.png</iconUrl>
     <licenseUrl>https://github.com/kubernetes/kompose/blob/master/LICENSE</licenseUrl>
+    <copyright>(c) 2020 The Kubernetes Authors</copyright>
```

This should be:

```suggestion
    <copyright>&copy; 2021 The Kubernetes Authors</copyright>
```

Or

```suggestion
    <copyright><![CDATA[© 2021 The Kubernetes Authors]]></copyright>
```

The latter is most likely preferred in this case, as the former may not work as expected (not completely sure).